pkg/command/autoscaling/list_test.go

```diff
@@ -0,0 +1,95 @@
+// Copyright © 2020 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package autoscaling
+
+import (
+	"strings"
+	"testing"
+
+	"gotest.tools/assert"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8sfake "k8s.io/client-go/kubernetes/fake"
+	"knative.dev/client/pkg/util"
+	"knative.dev/kn-plugin-admin/pkg"
+
+	"knative.dev/kn-plugin-admin/pkg/testutil"
+)
+
+func TestAutoscalingListEmpty(t *testing.T) {
+	t.Run("no flags", func(t *testing.T) {
+		cm := &corev1.ConfigMap{
+			ObjectMeta: metav1.ObjectMeta{
```

Comment:
Maybe when it’s empty the result should be “No autoscaling info available” or similar instead of a list with headers...

Reply:
Thanks @maximilien, I will improve the output for empty configs.

Reply:
@maximilien and @zhanggbj Thanks, for e2e test case, yes, I missed it, I will add, but I saw @zhanggbj added the test function for autoscaling `func (et *e2eTest) knAdminAutoscaling(t *testing.T, r *test.KnRunResultCollector)` in her RP #16 , I think I should wait the PR to deliver first and then modify the same function to add tests for `list` command or I can add it later.

Reply:
@chaozbj make sense, so let's wait #16 merge first and then you can rebase and add e2e in this PR then.

/hold